spot-client/src/common/app-state/spot-tv/actions.js

```diff
@@ -41,3 +41,22 @@ export function setSpotTVState(newState) {
         newState
     };
 }
+
+/* eslint-disable jsdoc/require-description-complete-sentence */
+/**
+ * Extracts a phone number(if any) from the invites passed to the meeting view as part of the URL query and stores it in
+ * Spot TV state so that Spot Remotes can display it in the UI.
+ *
+ * @param {Array<{ type: string, number: string }>} [invites] - An array of invites as specified by Jitsi Meet iframe
+ * API.
+ * @returns {Function}
+ */
+export function storePhoneNumberFromInvites(invites) {
+    return dispatch => {
+        const phoneInvites = invites && invites.filter(invite => invite.type === 'phone');
+        const phone = phoneInvites && phoneInvites[0] && phoneInvites[0].number;
+
+        dispatch(setSpotTVState({ invitedPhoneNumber: phone }));
```

Comment:
I don't think this needs to be wrapped in an extra function because I see no async that warrants it.

Reply:
yeah, I was stupid thinking there's no other way for it to work 🤦‍♂ but actually it should work without the async just fine